hack/kind/main.go

```diff
@@ -0,0 +1,263 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// manifest is a kind cluster template
+// the explicit podSubnet definition can be removed as soon as https://github.com/kubernetes-sigs/kind/commit/60074a9e67ddc8d35d3468ab137358b62a4cf723
+// will be available in a released version of kind
+const manifest = `---
+kind: Cluster
+apiVersion: {{.APIVersion}}
+networking:
+  ipFamily: {{.IPFamily}}
+{{- if eq .IPFamily "ipv6" }}
+  podSubnet: "fd00:10:244::/56"
+{{- end}}
+nodes:
+  - role: control-plane
+{{- range .WorkerNames }}
+  - role: worker
+{{- end}}
+`
+
+var k kind
+
+type startArgs struct {
+	IPFamily string
```

Comment:
Why is this the only public field?

Reply:
Because it's the only one I am using in the template ...